php/php.editor/src/org/netbeans/modules/php/editor/lexer/GSFPHPLexer.java

```diff
@@ -65,6 +65,12 @@ public Token<PHPTokenId> nextToken() {
             PHPTokenId tokenId = scanner.nextToken();
             Token<PHPTokenId> token = null;
             if (tokenId != null) {
+                if (tokenId == PHPTokenId.PHP_CONSTANT_ENCAPSED_STRING) {
+                   PHP5ColoringLexer.LexerState state = scanner.getState();
+                   if (state.heredoc != null && state.heredoc.contains("CSS")) {
```

Comment:
Just thinking whether it should be `.contains()` or something more specific... :thinking: